worlds/messenger/docs/en_The Messenger.md

```diff
@@ -49,28 +49,28 @@ for it. The groups you can use for The Messenger are:
 ## Other changes
 
 * The player can return to the Tower of Time HQ at any point by selecting the button from the options menu
-  * This can cause issues if used at specific times. Current known:
-    * During Boss fights
-    * After Courage Note collection (Corrupted Future chase)
-      * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
-        is recommended to quit to title and reload the save
+    * This can cause issues if used at specific times. Current known:
```

Comment:
```suggestion
    * This can cause issues if used at specific times. Currently known areas:
```
Mirroring the other comment on this line, is mainly just an odd wording in my opinion
Not a vital change though